docs/qa/test_engineering.rst

```diff
@@ -30,19 +30,19 @@ The Normandy Firefox client queries the Normandy server for instructions and pic
 * QA: cmuresan
 * Automation: 
 * Test Engineer: b4hand
-* [Manual tests](https://testrail.stage.mozaws.net/index.php?/suites/view/232&group_by=cases:section_id&group_order=asc&group_id=16918)
+* _Manual tests: https://testrail.stage.mozaws.net/index.php?/suites/view/232&group_by=cases:section_id&group_order=asc&group_id=16918
```

Comment:
This doesn't work because there is no link between "Manual" and "tests". It's looking for a link with an ID of `Manual`.

The correct version would be either this inline version:

```rst
* `Manual tests <https://testrail.stage.mozaws.net/index.php?/suites/view/232&group_by=cases:section_id&group_order=asc&group_id=16918>` are maintained on Test Rail
```

OR this not-inline version:

```rst
* `Manual tests`_ are maintained on Test Rail

.. _Manual tests: https://testrail.stage.mozaws.net/index.php?/suites/view/232&group_by=cases:section_id&group_order=asc&group_id=16918
```